src/views/verify.tsx

```diff
@@ -1,9 +1,19 @@
 import { isValid, openAttestationVerifiers, verificationBuilder, VerificationFragment } from "@govtechsg/oa-verify";
+import { providers } from "ethers";
 import React, { useEffect, useState } from "react";
 import { FailedAlert, SucceedAlert } from "../components/alert";
 import { Status } from "../shared";
 
-type Network = "ropsten" | "homestead" | "rinkeby" | "goerli";
+enum SupportedNetwork {
+  ropsten = "Ropsten",
+  homestead = "Mainnet",
+  rinkeby = "Rinkeby",
+  goerli = "Goerli",
+  maticmum = "Mumbai",
```

Comment:
Are we only supporting Polygon Mumbai here or should we also support the Polygon mainnet too?

Reply:
yeah we can i guess, added